cmd/snap-discard-ns/snap-discard-ns.c

```diff
@@ -16,20 +16,23 @@
  */
 
 #include "../libsnap-confine-private/utils.h"
+#include "../libsnap-confine-private/locking.h"
 #include "../snap-confine/ns-support.h"
 
 int main(int argc, char **argv)
 {
 	if (argc != 2)
 		die("Usage: %s snap-name", argv[0]);
 	const char *snap_name = argv[1];
+
+	int snap_lock_fd = sc_lock(snap_name);
+	debug("initializing mount namespace: %s", snap_name);
 	struct sc_ns_group *group =
 	    sc_open_ns_group(snap_name, SC_NS_FAIL_GRACEFULLY);
 	if (group != NULL) {
-		sc_lock_ns_mutex(group);
```

Comment:
It seems `cmd/snap-confine/ns-support.h` still has a reference in the header to this one?